server/controllers.js

```diff
@@ -0,0 +1,12 @@
+const Products = require('../db/Products.js');
+
+module.exports = {
```

Comment:
Put function getProducts above module.exports and then have module.exports = {getProducts} for better organization